### PR TITLE
feat(projectcard component): i have incorporated a more isomorphic de…

### PR DESCRIPTION
**Pull request by arunpariyar**
…sign for the project cards

BREAKING CHANGE: It is specific to the mock data and project card component

### CODE REVIEW

File: src/types/Project.tsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
...
-  title: string,
-  create_date: Date,
-}
\ No newline at end of file
+  id: number;
+  author: string;
+  title: string;
+  create_date: Date;
+  intro: string;
+  tech: string[];
```
Comment (n4mr3g) [inline src/types/Project.tsx]: maybe tech**s** (in plural) could be a better name, as it is an array? or something else more descriptive, like 'techStacks'

File: src/types/Project.tsx [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
-  id: number,
-  author: string,
-  title: string,
-  create_date: Date,
-}
\ No newline at end of file
+  id: number;
+  author: string;
+  title: string;
+  create_date: Date;
```
Comment (n4mr3g) [inline src/types/Project.tsx]: maybe 'creationDate' or 'createdAt' would be better name options

File: src/styles/ProjectCard.module.css [hunk 3]
```diff
@@ -1,8 +1,38 @@
...
+  font-weight: 700;
+  margin-bottom: 8px;
+  font-family: "Jost", sans-serif;
+}
+
+.intro {
+  font-size: 12px;
+}
+
+.techAndDate {
```
Comment (n4mr3g) [inline src/styles/ProjectCard.module.css]: What about changing this selector to something like 'project-info', 'project-details'?

File: src/styles/ProjectCard.module.css
Comment (n4mr3g) [inline src/styles/ProjectCard.module.css:1]: We should probably keep CSS selectors in kebab-case

**Review by n4mr3g — CHANGES_REQUESTED** [review]

File: src/styles/ProjectCard.module.css
Comment (n4mr3g) [reply src/styles/ProjectCard.module.css:1]: Resolved: we allow camelCase due to how CSS modules work in Next.js

File: src/styles/ProjectCard.module.css (hunk 3 above)
Comment (arunpariyar) [reply src/styles/ProjectCard.module.css]: renamed it to simply details

File: src/types/Project.tsx (hunk 1 above)
Comment (arunpariyar) [reply src/types/Project.tsx]: renamed to techStacks :)

**Review by n4mr3g — APPROVED** [review]
LGTM

**Review by n4mr3g — APPROVED** [review]

**Comment by n4mr3g** [comment]
Solve merge conflicts

**Pull request merged**